scripts/py_matter_yamltests/matter_yamltests/parser.py

```diff
@@ -1223,7 +1223,8 @@ def _config_variable_substitution(self, value):
             # But some other tests were relying on the fact that the expression was put 'as if' in
             # the generated code and was resolved before being sent over the wire. For such
             # expressions (e.g 'myVar + 1') we need to compute it before sending it over the wire.
-            tokens = value.split()
+            delimiter_regex = "(\ |\+|\-|\*|\/|\%)"
```

Comment:
This should probably include `\(` and `\)` in the regex, so that things like `(myVariable + 4)/2` will work correctly.